spec/controllers/expenses_additional_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ExpensesAdditionalController, type: :controller do
+  let(:step) { assigns(:step) }
+  let(:step_class) { ExpensesAdditional }
+  before {}
```

Comment:
surprised we don't need a `current_snap_apploication` set for this page to render?